charts/airflow/templates/worker/worker-statefulset.yaml

```diff
@@ -162,7 +162,39 @@ spec:
             {{- else }}
             - "exec airflow celery worker"
             {{- end }}
-          {{- if $volumeMounts }}
+          {{- if .Values.worker.livenessProbe.enabled }}
+          livenessProbe:
+            initialDelaySeconds: {{ .Values.worker.livenessProbe.initialDelaySeconds }}
+            periodSeconds: {{ .Values.worker.livenessProbe.periodSeconds }}
+            timeoutSeconds: {{ .Values.worker.livenessProbe.timeoutSeconds }}
+            failureThreshold: {{ .Values.worker.livenessProbe.failureThreshold }}
+            exec:
+              command:
+              - celery
+              - --app
+              - airflow.executors.celery_executor.app
+              - inspect
+              - ping
+              - -d
+              - celery@${HOSTNAME}
+          {{- end }}
+          {{- if .Values.worker.readinessProbe.enabled }}
+          readinessProbe:
```

Comment:
I am not sure it makes sense to include a `readinessProbe` on a worker, as workers don't serve any traffic (in an HTTP service sense).

Read more about ["readiness probes" in the official docs](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-readiness-probes).

___NOTE:__ I still like the idea of a `livenessProbe` to restart unhealthy workers_

Reply:
@mtparet I think its best to remove it